### DISCUSSION TITLE
Not showing file/folder with "cache" in the name

### DISCUSSION THREAD

**Discussion by Guiguiprim** [nvim-tree/nvim-tree.lua#1442] [Q&A] [answered]
I realized that NvimTree  does not show files/folders with `cache` in the name in my project.

I tried to found a setting that could be linked to this [here](https://github.com/kyazdani42/nvim-tree.lua/blob/master/doc/nvim-tree-lua.txt), but did not found any. I'm using `LunarVim` which might have added a custom setup responsible for this behavior. 

Thanks in advance for any pointers you might have on this.

**Comment by kyazdani42** [comment, marked as answer]
check if
- cache is gitignored
- cache matches in filters.custom
You can toggle ignored and custom filters in nvim-tree, i'm not sure what the bindings for LunarVim are though.

**Reply by Guiguiprim** [reply]
So the setting I was looking for was `filters.custom`. Thanks.

And indeed in the version of LunarVim I'm using, it contains `.cache`. They improved it into `\\.cache` in new versions apparently. 
Removing it did fix my issue !
